### ISSUE TITLE
Library Explorer Covers do not have alt attributes

### ISSUE THREAD

**Issue by bpmcneilly** [internetarchive/openlibrary#4938]
The covers of books in the library explorer do not include `alt` attributes. Currently, the `img` elements have `title` attributes with the book's name.

This issue was uncovered as a part of #4640

### Evidence / Screenshot (if possible)
![The library explorer with Firefox's dev tools open & displaying a lack of alt attribute on book cover img elements](https://user-images.githubusercontent.com/3421881/112917719-1ffd2c80-90b8-11eb-880f-805204ca1f49.png)

### Relevant url?
https://openlibrary.org/explore

### Steps to Reproduce

1. Open the Library Explorer Page
2. Inspect an `img` element with `class="cover"`
3. Verify the `img` elements contain `title` attributes that match the `title` attributes of their parent `a` element.

### What actually happened after these steps? What did you expect to happen?
* Actual: `img` elements lack `alt` attributes as required by the HTML spec. This causes no alternative to be rendered for both this `img` for the parent link.
* Expected: All `img` elements have `alt` attributes with relevant values.

### Details

- **Logged in (N)?** 
- **All Browsers & Operating Systems** 
- **Environment: Prod**

### Proposal & Constraints
* The book cover links take users to the page for the relevant book, so the link text must inform them where they will be taken
* Often, book titles may not be extremely obvious. So retaining the title attributes somewhere may be helpful for users who will have title attributes rendered to them
    * This is typically on mouse hover, but not keyboard focus, in browser that support the feature
    * An example of this is in the screenshot above. The title of the book with its code highlighted is "Environmental Science", but this text is not easily visible on the cover, instead there are a series of trees.
    * I'd recommend keeping the `title` attribute on the `a` element & transitioning the title attribute on the `img` element to become an `alt` attribute

### Related files
`FlatBookCover.vue`

**Comment by Yashs911** [comment]
@bpmcneilly I would like to work on this issue.
